pyscriptjs/src/main.ts

```diff
@@ -179,6 +179,8 @@ export class PyScriptApp {
         // lifecycle (6.5)
         this.plugins.afterSetup(runtime);
 
+        //Refresh module cache in case plugins have modified the filesystem
+        runtime.interpreter.runPython(`import importlib; importlib.invalidate_caches()`)
```

Comment:
I would still suggest for Pyodide:
```suggestion
        let importlib = runtime.interpreter.pyimport("importlib"); 
        importlib.invalidate_caches();
```
since this doesn't pollute the globals. 

But if you are trying to support other runtimes maybe it's better to stick with something more portable? If you factor out `invalidate_caches` as a separate method, then maybe you could check if `pyimport` is available and use this method if so otherwise fall back to the other approach. Or maybe you should just have a persistent `runPythonPrivate` that runs things in a separate private context that won't pollute the user globals.